bot.py

```diff
@@ -47,8 +48,7 @@ def getRoleByName(guild, name):
 
 # Nicer than a dict for storing info about the town
 class TownInfo:
-    def __init__(self, ctx, document):
-        guild = ctx.guild
+    def __init__(self, guild, document):
```

Comment:
why swap out ctx for just a guild alone?

Reply:
ctx is not actually required. I need to call this from the timer function which does not have a ctx

Reply:
I need to call this from the timer function because the timer function calls endGameInternal.